utils/validation_schemas/projects-data.js

```diff
@@ -0,0 +1,65 @@
+const joi = require("joi");
+const validationRules = require("./validation-rules");
+const errors = require("../../configuration/errors");
+
+const projectDataSchema = joi.object({
+    name: joi.string().required().messages({
+        "string.empty": errors.PROJECT_NAME_NOT_PROVIDED,
+        "string.required": errors.PROJECT_NAME_NOT_PROVIDED,
+    }),
+    description: joi.string().required().messages({
```

Comment:
I think joi will automaticly return error "description missing" if you just put required(). I think you don't need to specify error messages. But check it